src/components/LHNOptionsList/OptionRowLHN.tsx

```diff
@@ -113,7 +113,7 @@ function OptionRowLHN({reportID, isFocused = false, onSelectRow = () => {}, opti
     const report = ReportUtils.getReport(optionItem.reportID ?? '');
     const isStatusVisible = !!emojiCode && ReportUtils.isOneOnOneChat(!isEmptyObject(report) ? report : null);
 
-    const isGroupChat = optionItem.type === CONST.REPORT.TYPE.CHAT && optionItem.chatType && !optionItem.isThread && (optionItem.displayNamesWithTooltips?.length ?? 0) > 2;
+    const isGroupChat = optionItem.type === CONST.REPORT.TYPE.CHAT && !optionItem.chatType && !optionItem.isThread && (optionItem.displayNamesWithTooltips?.length ?? 0) > 2;
```

Comment:
Should we use strict type comparison  - `optionItem.chatType !== ''`

Reply:
actually it will create a TS error because `chatType` is strictly typed as a union of certain values and there is no `''` value but I think in that case we should add `''` to available values as this is what can happen :D

Reply:
I see. perhaps we can check for null in this case?

Reply:
@kubabutkiewicz ^
last commit to add `''` looks weird

Reply:
why is that? its adding that for `chatType` empty string is a possible value

Reply:
yeah , you are right, I will go back to original solution as its also cause typecheck failings in other places